src/bin/composer-requirements.py

```diff
@@ -12,12 +11,14 @@ def merge_requirements(env_data, local_data):
 
     for package in env_require:
         if package in local_data['require'].keys():
-            local_data['require'][package]=env_require[package]
+            local_data['require'][package] = env_require[package]
             print('Found {0}: Replacing with {1}'.format(package, env_require[package]))
+        local_data['require'].update({package: env_require[package]})
```

Comment:
Doesn't this also do the job of the previous line (adding if the key isn't there, and updating the value if it is) ?

Reply:
Oh right! `update()` doesn't just append keys 🤯 I amended the commit.